lib/src/toast.dart

```diff
@@ -109,8 +111,10 @@ class BotToast {
         onClose: onClose,
         onlyOne: onlyOne,
         crossPage: crossPage,
-        title: (_) => Text(title),
-        subtitle: subTitle == null ? null : (_) => Text(subTitle),
+        title: (_) => Text(title, style: titleStyle),
+        subtitle: subTitle == null
+            ? null
+            : (_) => Text(subTitle, style: subTitleStyle ?? titleStyle),
```

Comment:
I added that for the sake of uniformity, because most of the time the subtitle style will likely be the same as the title style. So in this case, the user can just provide the title style and it will be applied to both of them